src/wallet/rpcwallet.cpp

```diff
@@ -1817,7 +1831,7 @@ UniValue gettransaction(const UniValue& params, bool fHelp)
     WalletTxToJSON(wtx, entry);
 
     UniValue details(UniValue::VARR);
-    ListTransactions(wtx, "*", 0, false, details, filter);
+    ListTransactions(wtx, "*", 0, false, details, filter,"");
```

Comment:
The "pick-all" value for account parameter seems to be *. Why for address, you chose "" instead?